CNAME

```diff
@@ -1 +0,0 @@
-omemo.top
```

Comment:
@ReneVolution that's right, and I hoped that github would allow me to squash all my commits when opening the PR -- I guess I'll create an issure for that at github :D